crates/bin/prove_block/src/main.rs

```diff
@@ -24,15 +22,15 @@ fn init_logging() {
 }
 
 #[tokio::main]
-async fn main() -> Result<(), Box<dyn Error>> {
+async fn main() {
     init_logging();
 
     let args = Args::parse();
 
     let block_number = args.block_number;
     let layout = LayoutName::all_cairo;
 
-    let result = prove_block::prove_block(block_number, &args.rpc_provider, layout).await;
-    let (_pie, _output) = result.map_err(debug_prove_error).expect("Block could not be proven");
-    Ok(())
+    let result = prove_block::prove_block(block_number, &args.rpc_provider, layout, false).await;
```

Comment:
@shamsasari why are you setting the full output as false?
It makes sense to have it in true as this is the default use case Starkware need for the SNOS.